packages/web-components/src/components/rux-tree-node/rux-tree-node.tsx

```diff
@@ -67,7 +83,7 @@ export class RuxTreeNode {
         this.setSelected(newValue)
     }
 
-    @Listen('keydown', { passive: true })
+    @Listen('keydown', { passive: false })
```

Comment:
do you know what the issue with this was?

Reply:
Yes, if you have any scroll, and press the right or down key arrow, you do what that key should do in the component and scroll at the same time.

Basically if it's set to true the e.preventDefault does not work and you'll get an error saying something like "preventDefault won't work on passive events"